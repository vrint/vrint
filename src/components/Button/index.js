import Button from './Button'

/* istanbul ignore next */
// VBtn.install = function install (Vue) {
//   Vue.component(VBtn.name, VBtn)
// }

export default Button
