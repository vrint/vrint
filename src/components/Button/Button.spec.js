import { test } from '@util/testing'
import Vue from 'vue';
import Button from './Button';

test('Button.js', ({ mount, compileToFunctions }) => {
  it('should render component and match snapshot', () => {
    const wrapper = mount(Button);

    expect(wrapper.html()).toMatchSnapshot();
  });
});
