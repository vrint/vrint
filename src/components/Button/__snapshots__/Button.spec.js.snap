// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button.js should render component and match snapshot 1`] = `

<button type="button"
        class="pt-button"
>
  Default button
</button>

`;
